src/python/library/tritonclient/grpc/__init__.py

```diff
@@ -2259,6 +2266,11 @@ def _process_response(self, responses):
                     result = InferResult(response.infer_response)
                 self._callback(result=result, error=error)
         except grpc.RpcError as rpc_error:
+            # On GRPC error, refresh the active state to indicate if the stream
+            # can still be used. The stream won't be closed here as the thread
+            # executing this function is managed by stream and may cause
+            # circular wait
+            self._active = responses.is_active()
```

Comment:
I couldn't find where the `is_active()` is set in the GRPC frontend. Is this set in the frontend or something that the GRPC detects by itself?